Project.toml

```diff
@@ -48,42 +48,42 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 ZipFile = "a5390f91-8eb1-5f08-bee0-b1d1ffed6cea"
 
 [compat]
+julia = "1"
 AbstractPlotting = "^0.9.16"
 BinaryProvider = "0.3, 0.4, 0.5"
 ColorSchemes = "3.5"
-ColorTypes = "0.8, 0.9"
+ColorTypes = "0.9"
 ColorVectorSpace = "0.8"
-Colors = "0.9, 0.10, 0.11"
+Colors = "0.11"
 DataFrames = "0.20"
-Distributions = "0.21, 0.22"
-Documenter = "0.23, 0.24"
+Distributions = "0.22"
+Documenter = "0.24"
 FFMPEG = "0.2, 0.3"
 FFTW = "1"
 FileIO = "1.1"
-FixedPointNumbers = "0.6, 0.7, 0.8"
+FixedPointNumbers = "0.8"
 GDAL = "1"
-GeometryTypes = "0.7, 0.8"
+GeometryTypes = "0.8"
 HTTP = "0.8"
 Highlights = "0.4"
 ImageCore = "0.8"
 ImageFiltering = "0.6"
-ImageMagick = "0.7, 1.1"
+ImageMagick = "1.1"
 ImageTransformations = "0.8"
 JSON = "0.21"
-KernelDensity = "0"
-MakieLayout = "0"
+KernelDensity = "0.5.1"
+MakieLayout = "=0.3.0"
```

Comment:
```suggestion
KernelDensity = "^0.5.1"
MakieLayout = "0.3"
```

Reply:
We should force MakieLayout to a single version, since it keeps breaking, since the changes in MakieLayout aren't breaking, but visually different...

Reply:
As long as it's a temporary fix it should be fine...in the long term this gets kinda painful, but we shouldn't have to worry about that :D